snapcraft/plugins/rust.py

```diff
@@ -102,6 +103,40 @@ def build(self):
             cmd.append(' '.join(self.options.rust_features))
         self.run(cmd, env=self._build_env())
 
+    def _write_config(self):
+        if not self.project.is_cross_compiling:
+            return
+
+        # Cf. http://doc.crates.io/config.html
+        with suppress(FileExistsError):
+            os.mkdir(os.path.join(self.builddir, '.cargo'))
+        with open(os.path.join(self.builddir, '.cargo', 'config'), 'w') as f:
```

Comment:
a nit, I would put the cargo_dir in a variable, so there's no need to do the os.path.join again.